requirements.txt

```diff
@@ -12,3 +12,4 @@ six
 setuptools>=18.0
 pyyaml
 xarray
+statsmodels
```

Comment:
Is statsmodels a stable enough/supported enough library that it is OK to add it as a dependency @mcflugen